modules/mobile/spec/request/immunizations_request_spec.rb

```diff
@@ -422,5 +422,50 @@
         )
       end
     end
+
+    describe 'vaccines' do
```

Comment:
Could context blocks be used here?

```ruby
describe 'vaccine group name and manufacturer population' do
  let(:immunizations_request) do
    VCR.use_cassette('lighthouse_health/get_immunizations', match_requests_on: %i[method uri]) do
      get '/mobile/v0/health/immunizations', headers: iam_headers, params: nil
    end
  end
  
  context 'when the vaccine record exists' do
    let!(:vaccine) { create(:vaccine, cvx_code: 140, group_name: 'COVID-19', manufacturer: 'Moderna') }

    it 'uses the vaccine group name and manufacturer in the response' do
      immunizations_request
      expect(response.parsed_body['data'][2]['attributes']).to eq(...)
    end
  end
  
  context 'when the vaccine record does not exist' do
    it 'sets group name and manufacturer to nil' do
      immunizations_request
      expect(response.parsed_body['data'][2]['attributes']).to eq(...)
    end
  end
end
```

Reply:
Definitely. I may need to do some style adjustment. My last company had a rule against using `let!` and a strong preference for local variables in rspec over `let`. I have... mixed feelings about it.